easy.yaml

```diff
@@ -1,160 +1,171 @@
-#More general info here: https://docs.google.com/document/d/1r7qs1-MK7YbFf2d-mEUeTy2wHykIf1ALG9pLtVvUbSw/edit
-description: Easy/Open/Normal #please describe your options. Especially useful when you have multiple yamls for different occasions
-name: PleaseEnterNameHere #your name ingame, space and "_" gets replaced with a dash "-"
-glitches_required: none #there is also no_logic.
-item_placement: basic #this is based on Entrance Randomizer, which does not (yet?) support advanced
-map_shuffle: #to shuffle dungeon maps into the outside world and other dungeons, as well as other player's worlds in multiworld
+# What is this file?
+# This file contains options which allow you to configure your multiworld experience while allowing others
+# to play how they want as well.
+#
+# How do I use it?
+# The options in this file are weighted. This means the higher number you assign to a value, the more
+# chances you have for that option to be chosen. For example, an option like this:
+#
+# map_shuffle:
+#   on: 5
+#   off: 15
+#
+# Means you have 5 chances for map shuffle to not occur, and 15 chances for map shuffle to be turned on
+
+# I've never seen a file like this before. What characters am I allowed to use?
+# This is a .yaml file. You are allowed to use most characters.
+# To test if your yaml is valid or not, you can use this website:
+# http://www.yamllint.com/
+
+description: Farrak's Easy Example # Used to describe your yaml. Useful if you have multiple files
```

Comment:
I'd prefer to keep it as enter name here or something to that extent, as this is the most easily forgotten line

Reply:
Description updated in new commit.